src/main/java/org/spongepowered/api/effect/sound/SoundTypes.java

```diff
@@ -207,6 +207,8 @@
 
     public static final SoundType BLOCK_PORTAL_TRIGGER = DummyObjectProvider.createFor(SoundType.class, "BLOCK_PORTAL_TRIGGER");
 
+    public static final SoundType BLOCK_PUMPKIN_CARVE = DummyObjectProvider.createFor(SoundType.class, "BLOCK_PUMPKIN_CARVE");
```

Comment:
Carve or Carved?

Reply:
Carve. This is the sound that plays when an uncarved pumpkin turns into a carved pumpkin.